app.js

```diff
@@ -45,16 +51,23 @@ function displayPhrases() {
 
     for (const phrase of character.phrases) {
         // > create an <li> element
+        const li = document.createElement('li');
         // > set the text of the li to the phrase
+        li.textContent = phrase;
         // > append the <li> to the phrases list
+        if (li.textContent !== '') {
```

Comment:
Great work getting the extra credit in! Something to think about; Why are we doing the logic here? Since we have control of what is allowed to be pushed into the array, wouldn't we want to put the logic in a place that would stop empty strings from being pushed to the array in the first place? Let me know if that doesn't make sense, we can talk it over!